### PR TITLE
Supporting overriding supplier

### PR DESCRIPTION
**Pull request by KieranChandler**

### CODE REVIEW

File: src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx [hunk 1]
```diff
@@ -160,6 +191,26 @@ const EditExpense = ({
...
+            disabled={
+              // Ideally this would be handled in the API to avoid retrieving all of the contacts when a contact is not applicable for this transaction.
+              // However, since for simplicity this demo has the transactions stored locally in the browser, we are doing this check locally.
+              !isSupplierApplicable(expenseTransaction)
+            }>
+            {suppliers!.map((supplier) => (
+              <option
+                key={supplier.id}
+                value={supplier.id}
+                selected={supplier.id === expenseTransaction.contactRef?.id}
```
Comment (blundell89) [inline src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: React moans a little when you use `selected` on the option, can you use `value` on the `select` pls?

Comment (blundell89) [reply src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: `defaultValue`, even

Comment (KieranChandler) [reply src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: Its the same as the other fields in this page. Should I change those too to be consistent?

Comment (blundell89) [reply src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: I've updated those in my pr

Comment (blundell89) [reply src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: and that's merged now!

File: src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx [hunk 2]
```diff
@@ -192,6 +228,26 @@ const EditExpense = ({
           </select>
         </div>
 
+        <div className={styles.formRow}>
+          <label className={styles.inputLabel} htmlFor="account">Contact</label>
+          <select
+            id="account"
```
Comment (blundell89) [inline src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: shouldn't this be supplier?

Comment (KieranChandler) [reply src/pages/companies/[id]/expenses/[expenseId]/edit-expense.tsx]: Whoops, fixed

**Review by blundell89 — APPROVED** [review]

**Pull request merged**